cli.js

```diff
@@ -65,7 +69,25 @@ process.stdin
 		const geojson = JSON.parse(input)
 		const [west, south, east, north] = bbox(geojson)
 
-		const polylines = svgify(geojson, {projection: project})
+		const polylines = svgify(geojson, {
+			projection: project,
+			computeProps(feature) {
+				const {properties} = feature
+				const dataAttributes = Object.entries(properties).reduce((acc, entry) => {
+					const [key, value] = entry
+					if (!propertiesArray.includes(key)) return acc;
+
+					const dataKey = `data-${key.toLowerCase().replace('_', '-')}`
+					acc[dataKey] = value
+
+					return acc
+				}, {})
+				return Object.assign({
```

Comment:
Use `dataAttributes.className = 'shape'; return dataAttributes` to make sure `className` always has `shape` as value.